src/OMSimulatorLib/SystemWC.cpp

```diff
@@ -766,8 +768,10 @@ oms_status_enu_t oms::SystemWC::doStep()
 
     return oms_status_ok;
   }
+  else
+    return logError("Invalid solver selected");
 
-  return logError("Invalid solver selected");
+  return oms_status_error;
```

Comment:
Can we keep this line as it was before?

Reply:
this is fixed with https://github.com/OpenModelica/OMSimulator/pull/1163/commits/aa3bd100a20c837d57980029e11f6bc80c372c84